docs/src/tutorials/4_modifying.md

```diff
@@ -79,8 +79,11 @@ m.reactions["R_GLNS"].metabolites["M_co2_e"] = 1.0
 ```
 
 There are other functions that may be used to change the StandardModel in a
-more systematic way. See the documentation of [`add!`](@ref), [`rm!`](@ref),
-[`@add_reactions!`](@ref), and [`set_bound`](@ref) for examples.
+more systematic way. See the documentation of [`add_reaction!`](@ref), [`add_reactions!`](@ref),
+[`@add_reactions!`](@ref), [`add_metabolite!`](@ref), [`add_metabolites!`](@ref), [`add_gene!`](@ref), 
+[`add_genes!`](@ref), [`remove_reaction!`](@ref),[`remove_reactions!`](@ref), [`remove_metabolite!`](@ref),
+[`remove_metabolites!`](@ref), [`remove_gene!`](@ref), [`remove_genes!`](@ref), [`change_bound`](@ref),
+[`change_bound!`](@ref), [`change_bounds`](@ref), and [`change_bounds!`](@ref) for examples.
```

Comment:
this sounds a bit like "see this full list for examples".

Technically, can we link the whole documentation section?

Reply:
lol, yeah, this is literally all of them...

Reply:
but linking the docs would just generate an uglier list. I think this is fine and contained?

Reply:
No I thought something like
```
[The complete list can be found here.](functions#Modifications)
```
to link here https://lcsb-biocore.github.io/COBREXA.jl/stable/functions/#Model-variant-specifications or here https://lcsb-biocore.github.io/COBREXA.jl/stable/functions/#Analysis-modifications

I'm not sure if hash anchors are supported but they technically could be. They are generated for sure.

Reply:
Hmm, I'd be more in favor of directly listing them since those references are really long and forces the reader to navigate away from the page they are reading